records_mover/db/vertica/unloader.py

```diff
@@ -97,7 +101,13 @@ def unload_to_s3_directory(self,
         processing_instructions = unload_plan.processing_instructions
         try:
             s3_sql = self.aws_creds_sql(aws_creds.access_key, aws_creds.secret_key)
-            self.db.execute(s3_sql)
+            if isinstance(self.db, sqlalchemy.engine.Connection):
```

Comment:
eugh, i hate that this object is constructed with either an engine or connection, and we don't know which. would it be a lot of work to fix that?

Reply:
Fixed